plugins/adr/package.json

```diff
@@ -45,9 +45,13 @@
     "react-router-dom": "6.0.0-beta.0 || ^6.3.0"
   },
   "devDependencies": {
+    "@backstage/catalog-client": "workspace:^",
     "@backstage/cli": "workspace:^",
     "@backstage/core-app-api": "workspace:^",
     "@backstage/dev-utils": "workspace:^",
+    "@backstage/integration": "workspace:^",
+    "@backstage/plugin-catalog": "workspace:^",
+    "@backstage/plugin-permission-react": "workspace:^",
```

Comment:
looks like these new deps are no longer necessary with the latest changes?

Reply:
I have removed these three dependencies.